src/github.com/getlantern/eventual/eventual_test.go

```diff
@@ -28,6 +28,18 @@ func TestSingle(t *testing.T) {
 	assert.Equal(t, "hi", r, "Wrong result")
 }
 
+func BenchmarkGet(b *testing.B) {
+	v := NewValue()
+	go func() {
+		time.Sleep(20 * time.Millisecond)
+		v.Set("hi")
+	}()
+
+	for i := 0; i < b.N; i++ {
+		v.Get(20 * time.Millisecond)
+	}
+}
```

Comment:
I had a hunch it was a little expensive, so I created the benchmark.  Until now, we weren't really using it on anything critical path.  Even in Dial() it was cheap compared with all the other work, but now we can confidently use it liberally without worrying much about performance.